scripts/safe_template_linter.py

```diff
@@ -197,7 +197,7 @@ def print_results(self):
         """
```

Comment:
Maybe you could pull linter changes out to a separate PR at some point?  Thanks.